sap/stackstate_checks/sap/sap.py

```diff
@@ -197,57 +187,61 @@ def _collect_instance_processes_and_metrics(self, host_instances):
                 })
 
     def _collect_processes(self, instance_id, host_instance_proxy):
-        processes = host_instance_proxy.get_sap_instance_processes()
+        processes = {}
```

Comment:
May be this line is not needed. Since in the next line it will be overriden by the function call response.

Reply:
Fixed in new commit.